### DISCUSSION TITLE
How to use it

### DISCUSSION THREAD

**Discussion by RedStained** [JannesStroehlein/Toggle-Razer-Chroma-Visualizer#4] [Q&A] [answered]
Hi, you say to place the .exe in any directory, but I can't find any executable. I'm confused.

**Comment by RedStained** [comment, marked as answer]
Oh, never mind, I found it (on the right side of the page). I've never used github before, I'm sorry. Weird UI choice.

**Reply by JannesStroehlein** [reply]
Thanks for the heads up. I added a link to the README file (the "homepage" of the project) to navigate directly to the page where you can download the exe file. 
I hope this also can help others to find the download because I agree that GitHub can be difficult to understand for non developers.

**Reply by RedStained** [reply]
It certainly will! And thank you very much, your program works.
